compat/src/index.js

```diff
@@ -366,14 +364,14 @@ options.vnode = vnode => {
 /**
  * Deprecated way to control batched rendering inside the reconciler, but we
  * already schedule in batches inside our rendering code
- * @param {(a) => void} callback function that triggers the updatd
- * @param {*} [arg] Optional argument that can be passed to the callback
+ * @template Arg
+ * @param {(arg: Arg) => void} callback function that triggers the updatd
```

Comment:
Minor typo in the end. It was present before but maybe can be fixed Addy with this PR 🙂

Reply:
Oh yup! Good catch. Fixed :)